contracts/interfaces/IThirdwebRoyalty.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "./IThirdwebModule.sol";
+
+// Helper interfaces
+import "@openzeppelin/contracts/interfaces/IERC2981.sol";
+
+interface IThirdwebRoyalty is IThirdwebModule, IERC2981 {
```

Comment:
created this interface that any of the contracts that support royalties should implement